lib/betagouvbot/webhook.rb

```diff
@@ -23,11 +23,8 @@ def members
       date = params.key?('date') ? Date.iso8601(params['date']) : Date.today
```

Comment:
Maybe not related to this PR, but now that we're paying a dyno, we should just get rid of this endpoint and create a job class that get executed once a day.